pages/linux/repquota.md

```diff
@@ -0,0 +1,27 @@
+# repquota
+
+> Display a summary of existing file quotas for a filesystem.
+
+- Report on all existing quotas:
+
+`repquota -all`
+
+- Report all quotas, even if there is no usage:
```

Comment:
Report quota, even for users those who do not consume resources. How about - 
Verbose mode - report all quotas, even for users those who do not consume resources: